TTHAnalysis/macros/prepareEventVariablesFriendTreeEdge.py

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/env python
```

Comment:
There shouldn't be the need to duplicate almost the entirety of this file.

Can we discuss how to keep only one, and refactor away in other python files the specific parts of edge and multilepton?